containers/build-or-download-cvmfs-debs.sh

```diff
@@ -1,10 +1,12 @@
 #/bin/bash
 cvmfsversion=$1
-arch=$(dpkg --print-architecture)
-os=debian11
 
 apt-get update
-apt-get install -y wget
+apt-get install -y wget lsb-release
+
+arch=$(dpkg --print-architecture)
+os="$(lsb_release -si | tr [:upper:] [:lower:])$(lsb_release -sr)"
+
 if [ "$arch" = "ppc64el" ] || [ "$arch" = "arm64" ]
```

Comment:
Does it still need to build the deb packages from source for Arm?

Reply:
There are packages for Ubuntu, see https://cernvm.cern.ch/fs/, but I'm not sure if they work on Debian.